packages/near-contract-standards/src/fungible_token/core.ts

```diff
@@ -0,0 +1,55 @@
+import { AccountId, PromiseOrValue, Balance } from "near-sdk-js"
+import { Option } from "../non_fungible_token/utils"
+export interface FungibleTokenCore {
+    /**
+     * Transfers positive `amount` of tokens from the `env::predecessor_account_id` to `receiver_id`.
+     * Both accounts must be registered with the contract for transfer to succeed. (See [NEP-145](https://github.com/near/NEPs/discussions/145))
+     * This method must to be able to accept attached deposits, and must not panic on attached deposit.
+     * Exactly 1 yoctoNEAR must be attached.
+     * See [the Security section](https://github.com/near/NEPs/issues/141#user-content-security) of the standard.
+     *
+     * Arguments:
+     * @param receiver_id - the account ID of the receiver.
+     * @param amount - the amount of tokens to transfer. Must be a positive number in decimal string representation.
+     * @param memo - an optional string field in a free form to associate a memo with this transfer.
+     */
+    ft_transfer(receiver_id: AccountId, amount: Balance, memo: Option<String>);
```

Comment:
These probably should be:
```
    ft_transfer({receiver_id, amount, memo}: {receiver_id: AccountId, amount: Balance, memo: Option<String>});

```

Because Rust ft contract can be called with JSON args (tested with near-cli), in JS contract this is achieved by passing single args,

Reply:
Same for other `ft_*` methods